server/prisma/schema.prisma

```diff
@@ -45,8 +49,38 @@ model Credential {
   createdAt DateTime @default(now())
 }
 
-enum DatabaseType {
-  POSTGRES
+// TODO: dashes are not supported in enums
+// enum FhirKind {
+//   logical
+//   primitive-type
+//   complex-type
+//   resource
+// }
+
+enum FhirDerivation {
+  constraint
+  specialization
```

Comment:
Isn't it better to have upper case enum values?

Reply:
yes definitely, but then we need to change the actual `derivation` field from the StructureDefinition (which is lowercase). what do you think is best ?

Reply:
Hum, right, or we could capitalize the string in between (something like `derivation: resource.derivation.toUpperCase()`) but I'm not sure if we should do that or not... We don't want this .toUpperCase() at several places